test/cypress/integration/stripe-setup/update-org-details.cy.test.js

```diff
@@ -173,6 +181,19 @@ describe('The organisation address page', () => {
             cy.get('#address-postcode').should('have.value', invalidPostcode)
           })
       })
+
+      it('successfully submitting the form should redirect to the `add bank account details` page', () => {
+        cy.get('[data-cy=form]')
+          .within(() => {
+            cy.get('#address-postcode').clear()
+            cy.get('#address-postcode').type(validPostcode)
+            cy.get('[data-cy=continue-button]').click()
+
+            cy.location().should((location) => {
+              expect(location.pathname).to.eq(`/account/a-valid-external-id/your-psp/a-valid-credential-external-id/bank-details`)
```

Comment:
Shouldn't this be the `government entity document` page? Is this because of the test setup that the next page is bank details?

Reply:
Yeah - the test is set up like that.
I will look into changing this.

Reply:
I have updated this - took a bit more work then I thought.
I had to update the `getGatewayAccountStripeSetupFlagForMultipleCalls` function.